homeassistant/helpers/entity.py

```diff
@@ -151,7 +170,13 @@ def update_ha_state(self, force_refresh=False):
                 "No entity id specified for entity {}".format(self.name))
 
         if force_refresh:
-            self.update()
+            if hasattr(self, 'async_update'):
```

Comment:
yeah, the `update()` could be called outside from async loop and could not be use with other async api calls. `async_update()` is the developer Aware that this run inside async loop and he can use async feature. That is the big different.